### ISSUE TITLE
wifi rtl8192eu, ar9271 issues

### ISSUE THREAD

**Issue by noon92** [femtofox/femtofox#12]
rtl8192eu - not visible in iwconfig.
If 8812eus is connected first and is removed and replaced with 8192, connection continues (despite not showing in iwconfig). If however reboot is performed with 8192 connected (without 8812), no connection.

Same story with the ar9271, but will connect on boot (but not populate iwconfig),

With both, the interface (wlan0, wlan1, whatever) will appear, but remain empty

**Comment by noon92** [comment]
These errors are specific to 9271 - the 8912 has no similar errors:

```[  138.111711] ath: phy1: Failed to wakeup in 500us
[  138.122008] ath: phy1: Failed to wakeup in 500us
[  143.647226] usb 1-1.1: device descriptor read/64, error -110
[  167.487284] usb 1-1.1: device descriptor read/64, error -110
[  169.207188] ath9k_htc 1-1.1:1.0: ath9k_htc: Target is unresponsive
[  169.207279] ath9k_htc: Failed to initialize the device
[  171.287171] usb 1-1.1: Service connection timeout for: 256
[  171.287230] ath9k_htc 1-1.1:1.0: ath9k_htc: Unable to initialize HTC services
[  171.291327] ath9k_htc: Failed to initialize the device
```

**Comment by noon92** [comment]
@Ruledo 
Even with the wifi-mesh-control service disabled on boot, my ar9271 doesn't show up in iwconfig, but wlan0 does show up unpopulated. A connection was made though.
Exact same symptoms with my rtl8192eu.
```
femto@femtofox:~$ iwconfig
lo        no wireless extensions.

eth0      no wireless extensions.

wlan0     no wireless extensions.```

**Comment by Ruledo** [comment]
I believe this is external usb hub issue or bad cable.

**Comment by noon92** [comment]
I don't think so. The issue happens direct from OTG, and is consistently present with some chipsets but not with others (rtl8188eus)

**Comment by joshbowyer** [comment]
I am using the AR9271 without issue.

**Comment by NomDeTom** [comment]
@noon92 has this been resolved? Can we close this?

**Comment by noon92** [comment]
Honestly, I haven't tested it further. We have modules that work, so I quit messing with it.
